### PR TITLE
some global cleanup

### PR DESCRIPTION
**Pull request by grimreaper**
Some small amount of global cleanup.

### CODE REVIEW

File: code/src/java/pcgen/output/model/DateModel.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
 	 */
 	public DateModel(Date d)
 	{
-		this.date = d;
+		this.date = d.clone();
```
Comment (grimreaper) [inline code/src/java/pcgen/output/model/DateModel.java]: FWIW this was a manual bug fix - nothing automated.

File: code/src/java/pcgen/gui2/tools/TipOfTheDayHandler.java [hunk 2]
```diff
@@ -109,15 +109,7 @@ public synchronized void loadTips()
...
-			}
-			catch (FileNotFoundException e)
-			{
-				if (Logging.isDebugMode())
-				{
-					Logging.debugPrint("Unable to load tips file " + path, e); //$NON-NLS-1$
-				}
-			}
-			catch (IOException e)
+			} catch (IOException e)
```
Comment (cpmeister) [inline code/src/java/pcgen/gui2/tools/TipOfTheDayHandler.java]: Formatting needs to be fixed

Comment (grimreaper) [inline code/src/java/pcgen/gui2/tools/TipOfTheDayHandler.java]: fixed

File: code/src/java/pcgen/core/utils/SortedProperties.java [hunk 3]
```diff
@@ -75,12 +75,7 @@ public void mystore(final FileOutputStream out, final String header)
 			}
 
 			bw.flush();
-		}
-		catch (UnsupportedEncodingException ex)
-		{
-			Logging.errorPrint("Error writing to the options.ini file: ", ex);
-		}
-		catch (IOException ex)
+		} catch (IOException ex)
```
Comment (cpmeister) [inline code/src/java/pcgen/core/utils/SortedProperties.java]: Fix the formatting

File: code/src/java/pcgen/io/ExportHandler.java [hunk 4]
```diff
@@ -347,19 +347,12 @@ private void exportCharacterUsingFreemarker(PlayerCharacter aPC, BufferedWriter
...
 			throw new ExportException(exc, message + " : " + exc.getLocalizedMessage());
-		}
-		catch (TemplateException e)
-		{
-			String message = "Error exporting character using template " + templateFile;
-			Logging.errorPrint(message, e);
-			throw new ExportException(e, message + " : " + e.getLocalizedMessage());
-		}
-		finally
+		} finally
```
Comment (cpmeister) [inline code/src/java/pcgen/io/ExportHandler.java]: Here too

**Pull request merged**